### DISCUSSION TITLE
May I ask if it's possible not to use FlyskyHallStick? Replace with the original potentiometer

### DISCUSSION THREAD

**Discussion by zhuwei** [EdgeTX/edgetx#4541] [Q&A] [answered]
"The code includes an automatic detection of FlySky digital hall gimbals on power-up of TX16S/T16/T18 radios and no special build-time option is required. If after initial 70 ms the FlySky gimbals are not detected, the code reverts to sampling analog sticks."
What is the GPIO corresponding to the analog joystick? Where exactly is the code set?

**Comment by richardclli** [comment]
Try read the CMakelists.txt in targets, you need to have this line to enable Flysky Gimbal detection:

https://github.com/EdgeTX/edgetx/blob/3722de987f8e19dc41a61bdfd2f62fa591ecbd6c/radio/src/targets/horus/CMakeLists.txt#L86C5-L86C5

**Reply by zhuwei** [reply]
Thank you, but I would like to know what is the default GPIO for the analog joystick?

**Reply by richardclli** [reply]
The GPIO is radio specific, check out the definitions in hal.h

**Reply by zhuwei** [reply]
Thank you very much

**Comment by pfeerick** [comment]
There is also more information on the Flysky Hall Sticks mod here => https://github.com/EdgeTX/edgetx/wiki/Flysky-Hall-Sticks-Mod

**Reply by zhuwei** [reply]
This should explain how to use Flysky Hall Sticks, but I don't want to use it

**Reply by pfeerick** [reply, marked as answer]
Ah, sorry, I thought you were after the GPIOs for the hall sticks, which is actually mentioned somewhat obliquely on that page for some TXs... i.e. UART4 TX and RX, which equates to PA0/PA1. In that case, you're after this => https://github.com/EdgeTX/edgetx/blob/0506841f25624d13c487806ec9ec31563070806d/radio/src/targets/horus/hal.h#L291-L294

Unless you're not asking about a X10 family transmitter, as you never mentioned what transmitter this is in relation to ;)

**Reply by zhuwei** [reply]
Thank you very much
